docs/deploy/index.rst

```diff
@@ -46,20 +54,105 @@ on a Linux based embedded system such as Raspberry Pi:
     cmake ..
     make runtime
 
-Note that we type `make runtime` to only build the runtime library.
+Note that we type ``make runtime`` to only build the runtime library.
+
+It is also possible to cross compile the runtime. Cross compiling
```

Comment:
It would be helpful to describe why you would want to cross compile the runtime.

Reply:
i added a few lines on that and also i added a diagram the visually describes the process, i think that 
improved the clarify compared to describing it by words

this is how that is rendered in the documentation

![image](https://user-images.githubusercontent.com/3264637/116707916-7b7c4c00-a9cf-11eb-902f-c8aee573ee99.png)

please let me know if it is ok to include the figure in ``deploy/images`` (i added that folder) in some other places figures
are directly included from github rather than from files stores in the repo.

Reply:
ok, if you are ok with the image i can send a PR there.

Reply:
https://github.com/tlc-pack/web-data/pull/5

Reply:
thank you @tqchen  for merging https://github.com/tlc-pack/web-data/pull/5 
i updated the url, the doc build looks good locally.